src/main/java/com/example/bookshop/dto/book/CreateBookRequestDto.java

```diff
@@ -1,21 +1,24 @@
-package com.example.springfirsthw.dto;
+package com.example.bookshop.dto.book;
 
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
     @NotNull(message = "can't be null")
+    @Size(min = 1, max = 100)
     private String title;
-    @NotNull(message = "can't be null")
+    @NotNull(message = "author value can't be null")
     private String author;
-    @NotNull(message = "can`t be null")
+    @Size(min = 10, max = 13)
     private String isbn;
```

Comment:
try to use better validation for isbn. for example you can write your own `IsbnValidator` or use `@ISBN` from hibernate validation lib.